CMake/SlicerMacroBuildApplication.cmake

```diff
@@ -615,6 +652,31 @@ macro(slicerMacroBuildApplication)
           )
       endif()
 
+      if(EXISTS ${QT_LCONVERT_EXECUTABLE} AND NOT APPLE)
```

Comment:
To ensure preexisting cache value do not implicitly enable this code path (e.g after turning `Slicer_BUILD_I18N_SUPPORT` to `OFF`), I suggest to wrap this into `if(Slicer_BUILD_I18N_SUPPORT) / endif()`

Reply:
OK!